e2e/test/e2e_test.go

```diff
@@ -127,15 +127,17 @@ func test(t *testing.T, testFunc testFunc) {
 
 	if testFunc.TestOmniEVM != nil {
 		for _, chain := range network.Chains {
-			if chain.IsOmniEVM {
-				client, err := ethclient.Dial(chain.Name, chain.RPCURL)
-				require.NoError(t, err)
-
-				t.Run(chain.Name, func(t *testing.T) {
-					t.Parallel()
-					testFunc.TestOmniEVM(t, client)
-				})
+			if chain.ID != network.ID.Static().OmniExecutionChainID {
```

Comment:
Why not use the helper here?

e.g. 

`if !(netconf.IsOmniConsensus(network.ID, chain.ID)) {...}`

Reply:
I only added the helper when used more than 3 times